CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Husky Changelog
 
+## 0.8.0 2022-02-03
+
+## !!! Breaking Changes !!!
+
+- feat!: include individual batch sizes (#38)
```

Comment:
Is there any note we can add here to help the projects that use husky adjust to the breaking change?

Reply:
That's Basenji, Refinery, and Shepherd, right? And our team is making the changes for all of them right now. Not that it would hurt to have it, but I'm honestly more eager to have the version stamped so I don't have to edit my go.mod file. :)

Reply:
Yes. And OK.